src/Widgets/JobOverviewWidget/JobOverviewWidgetComponent.js

```diff
@@ -17,33 +17,39 @@ Scrivito.provideComponent('JobOverviewWidget', ({ widget }) => {
 
   return (
     <div className="row">
-      {
-        jobs.map((job, index) =>
-          <div key={ `${job.id()}${index}` } className="col-sm-6">
-            <Scrivito.LinkTag to={ job } className="box-card">
-              <Scrivito.BackgroundImageTag
-                tag="span"
-                className="box-image"
-                style={ {
-                  background: { image: job.get('image') },
-                } }
-              />
-              <span className="box-topic arrow-right">
-                <h3 className="h3">{ job.get('title') }</h3>
-                <span>
-                  <i
-                    className={ `fa ${job.get('location') ? 'fa-map-marker' : ''} fa-2x` }
-                    aria-hidden="true"
-                    title="location"
-                  />
-                  <span>{ job.get('location') }</span>
-                </span>
-                <i className="fa fa-angle-right" aria-hidden="true" />
-              </span>
-            </Scrivito.LinkTag>
-          </div>
-        )
-      }
+      { jobs.map((job, index) => <Job key={ job.id() } job={ job } index={ index } />) }
+    </div>
+  );
+});
+
+const Job = Scrivito.connect(({ job, index }) => {
```

Comment:
I think `Job` is not a good name, since it can be easily be confused with the `Job` obj class. How about `JobItem` (similar to `EventItem` of  `EventOverviewWidgetComponent`)?